### PR TITLE
Pt br fixes

### PR DESCRIPTION
**Pull request by PedroRajao**
Adds/fixes translations, placeholder's fixes.

### CODE REVIEW

File: translations/base-pt-BR.yaml [hunk 1]
```diff
@@ -527,31 +527,31 @@ buildings:
...
         default:
-            name: Energy Wire
-            description: Allows you to transport energy.
+            name: &wire Cabo de Energia
+            description: Permite transportar energia.
     advanced_processor:
         default:
-            name: Color Inverter
-            description: Accepts a color or shape and inverts it.
+            name: Inversor de Cor
```
Comment (MizardX) [inline translations/base-pt-BR.yaml:534]: The `&advanced_processor` anchor is missing here. It is aliased from line 806.
```suggestion
            name: &advanced_processor Inversor de Cor
```

File: translations/base-pt-BR.yaml [hunk 2]
```diff
@@ -527,31 +527,31 @@ buildings:
...
+            description: Aceita uma cor ou forma e a inverte.
     energy_generator:
-        deliver: Deliver
-        toGenerateEnergy: For
+        deliver: Entregar
+        toGenerateEnergy: Para
         default:
-            name: Energy Generator
-            description: Generates energy by consuming shapes.
+            name: Gerador de Energia
```
Comment (MizardX) [inline translations/base-pt-BR.yaml:540]: The `&energy_generator` anchor is missing here. It is aliased from line 810.
```suggestion
            name: &energy_generator Gerador de Energia
```

File: translations/base-pt-BR.yaml [hunk 3]
```diff
@@ -822,16 +829,13 @@ keybindings:
...
         switchDirectionLockSide: "Planejador: Mudar de lado"
-        pipette: Pipeta
-        menuClose: Close Menu
-        switchLayers: Switch layers
-        advanced_processor: Color Inverter
-        energy_generator: Energy Generator
-        wire: Energy Wire
+        pipette: Conta-Gotas
+        menuClose: Fechar Menu
+        switchLayers: Alternar Camadas
```
Comment (MizardX) [inline translations/base-pt-BR.yaml:838]: Duplicate key. `switchLayers:` already appears on line 798.

File: translations/base-pt-BR.yaml [hunk 4]
```diff
@@ -822,16 +829,13 @@ keybindings:
         placeInverse: Inverter orientação de esteira
         pasteLastBlueprint: Colar último projeto
         massSelectCut: Cortar área
-        exportScreenshot: Exportar base inteira como imagem
+        exportScreenshot: Exportar Base inteira como Imagem
```
Comment (MizardX) [inline translations/base-pt-BR.yaml:832]: Duplicate key. `exportScreenshot:` already appears on line 799.

File: translations/base-pt-BR.yaml [hunk 5]
```diff
@@ -822,16 +829,13 @@ keybindings:
...
         mapMoveFaster: Mover mais rápido
         lockBeltDirection: Ativar planejador de correia
         switchDirectionLockSide: "Planejador: Mudar de lado"
-        pipette: Pipeta
-        menuClose: Close Menu
-        switchLayers: Switch layers
-        advanced_processor: Color Inverter
-        energy_generator: Energy Generator
-        wire: Energy Wire
+        pipette: Conta-Gotas
```
Comment (MizardX) [inline translations/base-pt-BR.yaml:836]: Duplicate key. `pipette:` already appears on line 815.

**Comment by PedroRajao** [comment]
@MizardX thanks for the input! after fixing the previous errors I forgot to delete the old code, hence the duplicated keys

**Comment by tobspr** [comment]
Could you have a look at the conflicts? :)

**Comment by PedroRajao** [comment]
@tobspr Sure, I'll have a look  :)

**Comment by PedroRajao** [comment]
conflicts resolution done!

**Comment by tobspr** [comment]
Ty!

**Pull request merged**